src/OpenDebugAD7/OpenDebugAD7.csproj

```diff
@@ -49,6 +49,16 @@
 
   <Import Project="..\..\build\Debugger.PIAs.Portable.Packages.settings.targets" />
 
+  <!-- 
+    This ItemGroup is to override the packages that being depended on for the PackageReferences above. 
+    The issue is that Microsoft.VisualStudio.Shared.VSCodeDebugProtocol ->
```

Comment:
>Microsoft.VisualStudio.Shared.VSCodeDebugProtocol [](http://example.com/codeflow?start=22&length=49)

Since we own this one, should we just fix it?